### ISSUE TITLE
trustSrc() HTTP 404 error causing closed socket

### ISSUE THREAD

**Issue by BrianErikson** [7yl4r/the-oregon-trajectory#41]
`127.0.0.1 - - [31/Mar/2015 19:53:51] "GET /%7B%7B::trustSrc(ghWatchBtn)%7D%7D HT
TP/1.1" 404`

and 

``127.0.0.1 - - [31/Mar/2015 19:53:51] "GET /%7B%7B::trustSrc(ghForkBtn)%7D%7D HT
TP/1.1" 404``

This error is caused quite frequently when hosted by the standard `Python` server, which causes the socket connection to close prematurely. 

The section of problematic code is thrown [here](https://github.com/7yl4r/the-oregon-trajectory/blob/master/ng-modules/socialButtons/socialButtons.html#L4) and [here](https://github.com/7yl4r/the-oregon-trajectory/blob/master/ng-modules/socialButtons/socialButtons.html#L8)

**Comment by BrianErikson** [comment]
@7yl4r, do you know what this function does, or is supposed to do?

**Comment by 7yl4r** [comment]
yeah, these are the github "fork" and "star" buttons, and that trustSrc function is a hack I had to put in so that angular would let them work. 

I can look into it more, but I've been focusing on other things.
